elements/lisk-transactions/test/10_delegate_transaction.ts

```diff
@@ -31,7 +34,15 @@ describe('Delegate registration transaction class', () => {
 	let storeAccountFindStub: sinon.SinonStub;
 
 	beforeEach(async () => {
-		validTestTransaction = new DelegateTransaction(validDelegateTransaction);
+		validTestTransaction = new DelegateTransaction({
+			...validDelegateTransaction,
+			networkIdentifier:
+				protocolSpecDelegateFixture.testCases.input.networkIdentifier,
```

Comment:
in general, to avoid this nest object extraction you can destruct in describe 
```
const { networkIdentifier, account, transaction } = protocolSpecDelegateFixture.testCases.input;
```
?

Reply:
Fixed.